jwt-authorizer/templates/init.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+
+b64="base64 -w0"
+uname | grep -i darwin > /dev/null
+is_darwin_rc=$?
+
+if [[ $is_darwin_rc -eq 0 ]]; then
+    b64="base64"
+fi
+
+echo "Hostname:"
+read HOSTNAME
+
+echo "Kubernetes Namespace:"
+read NAMESPACE
```

Comment:
Namespace is handled in helm by doing {{ .Release.Namespace }} - don't pass this in as a parameter as it's a first class thing in helm (since the command line can set it)

Reply:
And you should add this to all your templates, otherwise I think this will install into the default namespace always?